src/components/text/Content.tsx

```diff
@@ -32,3 +32,7 @@ export const ContentTextLight = styled(ContentText)`
 export const ContentBackgroundLight = styled(ContentBackground)`
   font-weight: ${props => props.theme.fonts.lightFontWeight};
 `
+export const ContentTextBold = styled(Content)`
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  font-size: ${prop => prop.theme.fonts.defaultFontSize};
```

Comment:
font-size should be removed as the inherited font-size from `Content` is the same.

Reply:
@bahaaTuffaha please adjust this.

Reply:
as discussed, it is fine as it is